generate/testdata/snapshots/TestGenerateWithConfig-ClientGetterNoContext#01-testdata-queries-generated.go

```diff
@@ -0,0 +1,69 @@
+// Code generated by github.com/Khan/genqlient, DO NOT EDIT.
```

Comment:
I think this may also be an artifact of some testing you did -- can you try removing it and then seeing if snapshot tests still pass?  I could be wrong; if it's needed then no problem!

Reply:
Indeed. deleted.